build/logrotator.go

```diff
@@ -73,6 +75,23 @@ func (r *RotatingLogWriter) InitLogRotator(logFile string, maxLogFileSize int,
 		return fmt.Errorf("failed to create file rotator: %w", err)
 	}
 
+	// The selected compressor was validate by the caller, so we don't need
+	// to handle an unknown compressor here.
+	var c rotator.Compressor
+	switch logCompressor {
+	case "gzip":
+		c = gzip.NewWriter(nil)
```

Comment:
I think we should either have this as the `default` case or return an error if an unknown compressor name is passed in. Since this function is used in other projects.

Reply:
Good point, forgot that this was exported to `tapd` 😓